lib/lita/adapters/slack/slack_user.rb

```diff
@@ -30,18 +30,19 @@ def from_data_array(users_data)
         # @return [String] The user's email address, e.g. alice@example.com
         attr_reader :email
         # @return [Hash] The raw user data received from Slack, including many more fields.
-        attr_reader :metadata
+        attr_reader :profile
 
-        def initialize(id, name, real_name, metadata)
+
+        def initialize(id, name, real_name, profile)
           @id = id
           @name = name
           @real_name = real_name.to_s
-          @email = metadata['email'].to_s
-          @metadata = metadata
+          @email = profile ? profile['email'] : ''
+          @profile = profile
```

Comment:
The `users.list` endpoint does not include a `metadata` field for each user. ([source](https://api.slack.com/methods/users.list#responses))